ReleaseTooling/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -20,6 +20,19 @@ import Utils
 /// CocoaPod related utility functions. The enum type is used as a namespace here instead of having
 /// root functions, and no cases should be added to it.
 enum CocoaPodUtils {
+  /// The linkage type to specify for CocoaPods installation.
+  enum LinkageType {
+    /// Forced static libraries. Uses `use_modular_headers!` in the Podfile.
```

Comment:
Add "required for module map generation" to the comment, since otherwise this would be useless.

Reply:
Good call, thanks!